packages/common/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@sphereon/openid4vci-common",
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.ts",
+  "scripts": {
+    "preinstall": "npx only-allow pnpm",
```

Comment:
Already handled in the workspace package.json

Reply:
removed.